docs/content/reference/scanners/_index.md

```diff
@@ -1,47 +0,0 @@
----
-title: "Scanners"
-date: 2020-07-14T08:43:12+03:00
-draft: false
-weight: 20
-pre: '<i class="fas fa-network-wired"></i> '
-tags: ["reference"]
----
```

Comment:
Why did we drop this? We still have scanners and fingerprinters.

Reply:
Can we just split it up into separate pages and reference what we have?

Reply:
@VakarisZ  Could we take this offline so I can understand what changes we want to make and apply them?